Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -683,18 +683,30 @@ using FT0 = FT0s::iterator;
 namespace fdd
 {
 DECLARE_SOA_INDEX_COLUMN(BC, bc);                      //! BC index
-DECLARE_SOA_COLUMN(AmplitudeA, amplitudeA, float[4]);  //!
-DECLARE_SOA_COLUMN(AmplitudeC, amplitudeC, float[4]);  //!
+DECLARE_SOA_COLUMN(AmplitudeA_000, amplitudeA_000, float[4]);  //!
```

Comment:
Can you also take care of a description everywhere, after the //!

Reply:
Done